src/components/BooksContainer.js

```diff
@@ -1,14 +1,41 @@
+/* eslint-disable array-callback-return */
 /* eslint-disable react/jsx-props-no-spreading */
-import React from 'react';
-import { useSelector } from 'react-redux';
-import Book from './Book';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+// import Book from './Book';
 import AddBook from './AddBook';
+import { getBooks, deleteBook } from '../redux/books/booksSlice';
 
 const BooksContainer = () => {
   // const dispatch = useDispatch();
-  const { book } = useSelector((store) => store.books);
+  const { book, isLoading, error } = useSelector((store) => store.books);
+  const dispatch = useDispatch();
 
-  if (book === null) {
+  useEffect(() => {
+    dispatch(getBooks());
+  }, [dispatch]);
+
```

Comment:
- [ ] [OPTIONAL] Your `add` and `remove` functions are updating the api successfully however in order for the changes to be visible, the user must refresh the page which can negatively impact the user experience. I would suggest you to implement the changes so the page refreshes after the change is made.


https://user-images.githubusercontent.com/44798044/233763640-710e6663-22d6-4a83-8f94-dfed453cbd67.mov

Reply:
Hello @shella12 , 
thank you so much for the feedback, please can you help with a link, article or anything that can help me fix this reload to add items issue? I will be very grateful for that.